### PR TITLE
Update examples with imports

### PR DESCRIPTION
**Pull request by virajmparekh**
Variable ts1, ts2, and index are never defined in dataframe df. Should ts1 and ts2 be strings and can we get rid of the index variable?

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -247,11 +247,13 @@ class DataFrame(NDFrame):
 
     Examples
     --------
+    >>> import numpy as np
+    >>> import pandas as pd
```
Comment (jorisvandenbossche) [inline pandas/core/frame.py]: Those two are not needed (we assume those, to not have to repeat those in each docstring. Also the tutorial docs everywhere assume those two imports)

**Comment by jorisvandenbossche** [comment]
What do you mean with "variable ts1, ts2,"? 
In which docstring are they included? (I don't see them in the DataFrame one)

**Comment by TomAugspurger** [comment]
@jorisvandenbossche they are in the DataFrame class docstring

```
    Examples
    --------
    >>> d = {'col1': ts1, 'col2': ts2}
    >>> df = DataFrame(data=d, index=index)
    >>> df2 = DataFrame(np.random.randn(10, 5))
    >>> df3 = DataFrame(np.random.randn(10, 5),
    ...                 columns=['a', 'b', 'c', 'd', 'e'])

```

@vparekh94 it'd be nice to expand that a bit with text in between each explaining the data being passed in. Like

```
    Examples
    --------
    From a dictionary of arrays:

    >>> d = {'col1': ts1, 'col2': ts2}
    >>> df = DataFrame(data=d, index=index)

    From a 2-D numpy array:

    >>> df2 = DataFrame(np.random.randn(10, 5))

    Same, but setting the column names:

    >>> df3 = DataFrame(np.random.randn(10, 5),
    ...                 columns=['a', 'b', 'c', 'd', 'e'])

```

(might not have the newlines entirely correct there.

**Comment by jorisvandenbossche** [comment]
Oh, I was really blind .. :-)

But indeed, @TomAugspurger 's suggestion is the way to go!

**Comment by TomAugspurger** [comment]
@vparekh94 are you able to update this? You can push additional commits to this branch.

**Comment by jreback** [comment]
can you rebase and update?

**Comment by jorisvandenbossche** [comment]
@vparekh94 sorry for the oversight on out part, but this has been fixed in the meantime in another PR https://github.com/pandas-dev/pandas/pull/16487. 
Thanks for the contribution anyway!

**Pull request closed without merging**